### ISSUE TITLE
Course department component

### ISSUE THREAD

**Issue by panchi64** [uprm-inso4101-2023-2024-S1/semester-project-tutoring-app#29]

**Comment by panchi64** [comment]
Any progress so far? Anything I can help with?

**Comment by williammartinez10** [comment]
Yes, I completed the component. I put the file inside the "molecules" folder since it has a section designated for the course component (which I think should be inside the "atoms" folder, but that can be addressed later). For that, I just placed a dummy button that serves as a placeholder for the course component to be rendered.

**Comment by williammartinez10** [comment]
Created pull request.
